packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -16,6 +22,7 @@ export const colors: any = {
     gray: "#8A8A8A",
     grayBackground: "rgba(0, 0, 0, 0.04)",
     boxShadow: "rgba(0, 0, 0, 0.08)",
+    hover: "rgba(0,0,0,.3)",
```

Comment:
super nit, but can we align our spacing on these as they are all the same? IE - Space after comma

Reply:
sure